vitrina/orgs/services.py

```diff
@@ -40,62 +47,247 @@ class Action(Enum):
 
 class Role(Enum):
     COORDINATOR = Representative.COORDINATOR
-    MANAGER = Representative.MANAGER
+    MANAGER = Representative.MANAGER  # Visi Tvarkytojai
+    RESOURCE_MANAGER = "resource_manager"  # Ištekliaus Tvarkytojas
+    GLOBAL_MANAGER = "global_manager"  # Globalus Tvarkytojas (is staff)
```

Comment:
Would it also be beneficial to add these "resource_manager" & "global_manager" as enums themselves (like it is for coordinator & manager). If they are roles for the Representative that could be added, otherwise, maybe putting an enum to an enum is not the best idea :d